x-pack/plugins/observability_solution/uptime/common/rules/uptime_rule_field_map.ts

```diff
@@ -32,6 +32,10 @@ export const uptimeRuleFieldMap: FieldMap = {
     type: 'text',
     required: false,
   },
+  'error.stack_trace': {
```

Comment:
Does it mean you also added this field for uptime rules?

Reply:
no, but both relies on same feature, so mapping needs to be same. but we don't enhance uptime rules anymore

Reply:
Just to double-check, do you mean like having common mappings due to using a shared index behind the scenes for both?

Reply:
yes